h5grove/content.py

```diff
@@ -19,16 +19,24 @@
 
 
 class EntityContent:
+    """Base content for an entity."""
+
     type = "other"
 
     def __init__(self, path: str):
         self._path = path
+        """ Path in the file. """
 
     def metadata(self) -> Dict[str, str]:
+        """Entity metadata
+
+        :returns: {"name": str, "type": str}
```

Comment:
As discussed, signatures such as these should be handled with something more appropriate than Sphinx autodoc. I keep this for a future PR